dist.ini

```diff
@@ -47,9 +48,13 @@ repo = binary-com/perl-App-Config-Chronicle
 [PodCoverageTests]
 [Test::UnusedVars]
 [Test::ReportPrereqs]
+[Test::Perl::Critic]
+critic_config = t/rc/.perlcriticrc
 [SpellingCommonMistakesTests]
-[CopyFilesFromBuild]
-copy = Makefile.PL
+[CopyFilesFromRelease]
+filename = lib/App/Config/Chronicle.pm
```

Comment:
I tried this with `[CopyFilesFromBuild]` before, but dzil will report `[CopyFilesFromBuild] Cannot copy {{ $dist->main_module->name }} from build: file does not exist`

So Maybe it can  be used only in the phrase of `dzil new` . I didn't study it further.